src/components/modules/caret.ts

```diff
@@ -120,7 +109,7 @@ export default class Caret extends Module {
         return $.isEmpty(node) && !isLineBreak;
       });
 
-      if (nothingAtLeft && focusOffset === firstLetterPosition) {
+      if (nothingAtLeft && focusOffset===0) {
```

Comment:
Lint your code pls

Reply:
Hi, I have added the test cases can you please review them?